Tests/MongoSwiftBenchmarks/DocumentBenchmarks.swift

```diff
@@ -3,11 +3,11 @@ import Foundation
 import XCTest
 import Nimble
 
-let tweetFile = URL(fileURLWithPath: basePath + "tweet.json")
+let tweetFile = URL(fileURLWithPath: basePath + "/tweet.json")
```

Comment:
nits for the future, but let's use string interpolation as much as possible

Reply:
👍